src/components/Enrichment.styles.tsx

```diff
@@ -85,9 +85,39 @@ export const MetricTitle = styled.div`
     color: #777777;
     background: #f6f5f8;
     padding: 2px 8px;
+    // make this container horizontal
+    flex-direction: row;
+    display: flex;
+    justify-content: space-between;
+    // make it centered vertically
+    align-items: center;
 `;
 
 export const MetricValue = styled.div`
     padding: 2px 8px;
     font-weight: 500;
 `;
+
+export const InfoIcon = styled.div`
+    background: #777777;
```

Comment:
We probably need to think about themes and put all our colors in one file, for standard appearance throughout the extension